resources/kcp/charts/kyma-environment-broker/templates/deployment.yaml

```diff
@@ -396,21 +396,21 @@ spec:
               name: config-volume
             - mountPath: /swagger/schema
               name: swagger-volume
-          {{- if eq .Values.global.database.embedded.enabled false}}
-            - name: cloudsql-instance-credentials
-              mountPath: /secrets/cloudsql-instance-credentials
-              readOnly: true
-            - name: cloudsql-sslrootcert
-              mountPath: /secrets/cloudsql-sslrootcert
-              readOnly: true
-          {{- end }}
           {{- if .Values.broker.profiler.memory }}
             - name: keb-memory-profile
               mountPath: /tmp/profiler
               readOnly: false
           {{- end }}
+          {{- if and (eq .Values.global.database.embedded.enabled false) (eq .Values.global.database.cloudsqlproxy.enabled false)}}
```

Comment:
Can you please help me understand what is the difference between `.global.database.embedded.enabled` and `.Values.global.database.cloudsqlproxy.enabled`? afaict, they are both always used together as 
```
if and (eq .Values.global.database.embedded.enabled false) (eq .Values.global.database.cloudsqlproxy.enabled false)
```

Reply:
received response in different channel, sharing here for future reference
> Actually we think that there should still be a configurable possibility to connect to the CloudSQL instance with or without the cloudsql-proxy if not using embedded Postgres. We need this configurability because we want to drop cloudsql-proxy usage in a phased approach. Because embedded.enabled is disabled in all KCP environments, we need a config differentiator for using or not using cloudsql-proxy in GCP managed Postgres database.